### ISSUE TITLE
Source of 2D Detections provided by CLOCs

### ISSUE THREAD

**Issue by nazMahmoud** [pangsu0613/CLOCs#47]
Hello @pangsu0613!  For reproducing results on second, we need to use 3D detections from a pretrained SECOND model  and also 2D detections from Cascaded RCNN. You mention the following with regards to extracting 2D detections:

"_For this example, we use detections with sigmoid scores, you could download the Cascade-RCNN detections for the KITTI train and validations set from here file name:'cascade_rcnn_sigmoid_data_'"

My question is related to the model that predicted 'cascade_rcnn_sigmoid_data' 2D detections, is this model trained on **training+validation or only on training set** ? When I looked at [cascaded-rcnn repo](https://github.com/zhaoweicai/mscnn/blob/master/examples/kitti_car/fetch_cascade_mscnn_car_model.sh),  it only provides pretrained models on training and validation set. I do not think they provide models trained only on training set. Could you please clarify this ?

**Comment by Viczyf** [comment]
> Hello @nazMahmoud . The model that predicted 'cascade_rcnn_sigmoid_data' is trained on training (3712 examples) set. You are right, cascaded-rcnn repo doesn't provide that, so, I trained it by myself.

As the model is train on the train set, when predicting the score on train set will be higher than it on val set. Will it affect final detection performance?

**Comment by Viczyf** [comment]
Thanks for your reply. And I have another question. When you combine CT3D's 3D candiates with cascaded-rcnn. Do you use 100 candidates before final NMS?
I change the param _NMS_POST_MAXSIZE_ in rpn to 500 and use them as 3D candidates. But after fused with 2D candidates, the score decrease and the performance gets worse.
Do you have any suggestions on apply CLOCs to other 3D detectors?
